.github/ort.config.yml

```diff
@@ -0,0 +1,20 @@
+ort:
+  packageConfigurationProviders:
+  - type: OrtConfig
+  packageCurationProviders:
+  - type: OrtConfig
+  analyzer:
+    skipExcluded: true
+  advisor:
+    skipExcluded: true
+  reporter:
+    config:
+      CycloneDx:
+        output.file.formats: JSON
+      SpdxDocument:
+        creationInfo.comment: SPDX document created by the ORT.
```

Comment:
This information is already present inside `creators` in the correct "Tool:" syntax, so do we really need this additionally?